packages/pg/lib/stream.js

```diff
@@ -3,12 +3,12 @@
  * @returns {Duplex}
  */
 module.exports.getStream = function getStream(ssl) {
-  const net = require('net')
-  if (typeof net.Socket === 'function') {
-    return new net.Socket()
-  } else {
+  if (isCloudflareRuntime) {
```

Comment:
I think you forgot to call the fn here, but that's ok if you accept my memoization changes below